net/pfSense-pkg-freeradius2/files/usr/local/pkg/googleauth.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/local/bin/python2.7

```

Comment:
Some code is from here:
http://www.brool.com/post/using-google-authenticator-for-your-website/

License is  "CC0 1.0 Universal License"